pandas/core/groupby/groupby.py

```diff
@@ -2005,7 +2005,7 @@ def cumcount(self, ascending: bool = True):
 
         Essentially this is equivalent to
 
-        >>> self.apply(lambda x: pd.Series(np.arange(len(x)), x.index))
+            self.apply(lambda x: pd.Series(np.arange(len(x)), x.index))
```

Comment:
Does this still render as code when you build the docs?

Reply:
Good catch! 

---

Fixed in 4f1b138f4e35fdbd0766e40d8c54bbe9a71f1e71

Reply:
Is keeping this as it was not an option? I don’t think we add code-blocks to docstring normally since they are oriented more towards generated HTML / PDF output and aren’t super useful in say a shell

Reply:
The only solution that comes to my mind is to add ```# doctest: +SKIP``` to this.

Reply:
@jreback do you know how we typically handle these? I don't think we use code-block in docstrings do we?

Reply:
@WillAyd We actually do: (although I not sure if this is wanted)

https://github.com/pandas-dev/pandas/blob/cabc31a536ee548c959e1710fa4d10ed6b2fae74/pandas/core/accessor.py#L223-L266

Reply:
May be using triple backticks? Otherwise, a skip seems reasonable. Or may be @jorisvandenbossche has a better idea.